catalog/config-controller-monitoring/scripts/prometheus-ui.yaml.install-by-script

```diff
@@ -0,0 +1,35 @@
+# Promethus UI
+# https://cloud.google.com/stackdriver/docs/managed-prometheus/query#promui-deploy
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend
+  namespace: config-controller-monitoring
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      serviceAccountName: prometheus-config-controller
+      containers:
+      - name: frontend
+        image: "gke.gcr.io/prometheus-engine/frontend:v0.1.1-gke.0"
+        args:
+        - "--web.listen-address=:9090"
+        - "--query.project-id=project-id" # kpt-set: --query.project-id=${project-id}
```

Comment:
Will this still be rendered by kpt? I thought that since it wasn't suffixed by `.yaml`, it won't be considered.

Reply:
I fixed this in commit `df42054`, my mistake!

Reply:
Thanks! the new changes seem to include two files: the `.install-by-script` and the regular yaml file.

The install script only references the `.install-by-script`, and kpt only operations on files with the `.yaml`. suffix.

How will this work in practice? Are you trying to do something like:

1. have `kpt fn render` touch the `scripts/prometheus-ui.yaml` file (which should just happen)
2. run the install script
3. remove or rename the `prometheus-ui.yaml` file (necessary because kpt will try to apply otherwise)
3. `kpt live apply`

If so, I think this is close, but this file needs to be removed, and the apply script needs to switch to apply the `prometheus-ui.yaml` file.

Reply:
File has been removed.